### ISSUE TITLE
Error SyntaxError: Unexpected token ... using Node v4

### ISSUE THREAD

**Issue by km-tr** [facebookincubator/exerslide#3]
`exerslide build` was error when I tried `Getting started` on npm@2.15.8.

```
 ❯ exerslide build                                                                                                                                                                                                               
Error SyntaxError: Unexpected token ...
```

It has succeeded on npm@3.10.3.

**Comment by fkling** [comment]
Are you also using a different node version when you use npm@2.15.8? If yes which one?

**Comment by km-tr** [comment]
npm@2.15.8: v4.4.7
npm@3.10.3: v6.5.0

**Comment by fkling** [comment]
Then we are likely using a syntax feature that is not available in Node 4. I tried to avoid them but might have missed a place or two.

Will investigate, thank you!

**Comment by fkling** [comment]
Should be fixed with 40ce45b82fc078aedbfe30bffac74fc2246e49cc.

This problem didn't occur in tests because we where using babel-register for testing 😒 . I split the tests commands into two now, so no transformer is used for server side code.

I'll see that we get Travis CI set up for this repo in the next few days.

Thanks again for reporting this!
